package.json

```diff
@@ -27,39 +27,41 @@
         "node": ">=0.12"
     },
     "devDependencies": {
-        "@exabyte-io/code.js": "2023.11.1-0",
-        "@exabyte-io/eslint-config": "^2022.11.17-0",
+        "@exabyte-io/code.js": "git+https://github.com/Exabyte-io/code.js.git#update/SOF-7096",
+        "@exabyte-io/eslint-config": "^2023.8.29-1",
+        "@typescript-eslint/eslint-plugin": "^5.56.0",
+        "@typescript-eslint/parser": "^5.56.0",
         "chai": "^4.3.4",
         "chai-almost": "^1.0.1",
-        "eslint": "^7.32.0",
-        "eslint-config-airbnb": "^19.0.2",
-        "eslint-config-prettier": "^8.5.0",
-        "eslint-plugin-import": "^2.25.3",
-        "eslint-plugin-jsdoc": "^37.1.0",
-        "eslint-plugin-jsx-a11y": "^6.5.1",
+        "eslint": "7.32.0",
+        "eslint-config-airbnb": "19.0.2",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-import": "2.25.3",
+        "eslint-plugin-jsdoc": "37.1.0",
+        "eslint-plugin-jsx-a11y": "6.5.1",
+        "eslint-plugin-mui-path-imports": "0.0.15",
         "eslint-plugin-prettier": "^4.2.1",
-        "eslint-plugin-react": "^7.27.1",
-        "eslint-plugin-react-hooks": "^4.3.0",
-        "eslint-plugin-simple-import-sort": "^7.0.0",
+        "eslint-plugin-react": "7.30.0",
+        "eslint-plugin-simple-import-sort": "7.0.0",
+        "eslint-import-resolver-exports": "^1.0.0-beta.2",
```

Comment:
Can we replace these with the eslint plugin from npm?

Reply:
Am afraid we need all of this. From [README](https://github.com/Exabyte-io/eslint-config)

```
1. Install config with eslint and eslint plugins

npm install --save-dev \
  @exabyte-io/eslint-config \
  @babel/eslint-parser@7.16.3 \
  @babel/plugin-proposal-class-properties@7.16.0 \
  @babel/preset-env@7.16.4 \
  @babel/preset-react@7.16.7 \
  @babel/register@^7.16.0 \
  @babel/runtime-corejs3@7.16.8 \
  @typescript-eslint/eslint-plugin@5.9.1 \
  @typescript-eslint/parser@5.9.1 \
  eslint@7.32.0 \
  eslint-config-airbnb@19.0.2 \
  eslint-config-prettier@8.5.0 \
  eslint-import-resolver-exports@^1.0.0-beta.2 \
  eslint-import-resolver-meteor@^0.4.0 \
  eslint-import-resolver-node@^0.3.6 \
  eslint-plugin-import@2.25.3 \
  eslint-plugin-jsdoc@37.1.0 \
  eslint-plugin-jsx-a11y@6.5.1 \
  eslint-plugin-prettier@4.2.1 \
  eslint-plugin-react@7.30.0 \
  eslint-plugin-simple-import-sort@7.0.0 \
  eslint-plugin-mui-path-imports
  ```